app/controllers/jobseekers/profiles_controller.rb

```diff
@@ -4,31 +4,31 @@ class Jobseekers::ProfilesController < Jobseekers::BaseController
       title: "Personal details",
       key: "",
       link_text: "Add personal details",
-      page_path: -> { "" },
+      page_path: -> { "/" },
     },
     {
       title: "Job preferences",
       key: "",
       link_text: "Add job preferences",
-      page_path: -> { "" },
+      page_path: -> { "/" },
```

Comment:
Why this change sorry?

Reply:
i dunno, was from early on in coding task i think i change back to original